aws/resource_aws_s3_bucket_object_test.go

```diff
@@ -588,6 +588,29 @@ func testAccAWSS3BucketObjectCreateTempFile(t *testing.T, data string) string {
 	return filename
 }
 
+func TestAccAWSS3BucketObject_metadata(t *testing.T) {
+	rInt := acctest.RandInt()
+	var obj s3.GetObjectOutput
+
+	resource.Test(t, resource.TestCase{
```

Comment:
Since this pull request was written, parallelized testing is now available via [`resource.ParallelTest()`](https://godoc.org/github.com/hashicorp/terraform/helper/resource#ParallelTest)

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```